CGCharter-1727386911.html

```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
       or discriminate against any group participant or their employer.
     </p>
     <h2 id="chairs">
-      Chair Selection
+      Chairs
     </h2>
     <p>
-      Participants in this group choose their Chair(s) and can replace their
-      Chair(s) at any time using whatever means they prefer. However, if 5
-      participants, no two from the same organisation, call for an election,
-      the group must use the following process to replace any current Chair(s)
-      with a new Chair, consulting the Community Development Lead on election
-      operations (e.g., voting infrastructure and using <a href=
-      "https://tools.ietf.org/html/rfc2777">RFC 2777</a>).
+      The <a href="https://www.w3.org/Guide/chair/role.html">role of the Chair is described in the <a href="https://www.w3.org/Guide/">Art of Consensus</a>.
     </p>
-    <ol>
-      <li>Participants announce their candidacies. Participants have 14 days to
-      announce their candidacies, but this period ends as soon as all
-      participants have announced their intentions. If there is only one
-      candidate, that person becomes the Chair. If there are two or more
-      candidates, there is a vote. Otherwise, nothing changes.
-      </li>
-      <li>Participants vote. Participants have 21 days to vote for a single
-      candidate, but this period ends as soon as all participants have voted.
-      The individual who receives the most votes, no two from the same
-      organisation, is elected chair. In case of a tie, RFC2777 is used to
-      break the tie. An elected Chair may appoint co-Chairs.
-      </li>
-    </ol>
     <p>
-      Participants dissatisfied with the outcome of an election may ask the
-      Community Development Lead to intervene. The Community Development Lead,
-      after evaluating the election, may take any action including no action.
+      The Community Group participants elect (and/or other chairs appoint) no
+      less than One (1) and no more than Three (3) Chairs for the group at any
+      given time. The W3C Team Contact of the group SHOULD NOT be one of the
+      chairs at any point. Participation as Chair is afforded to the specific
+      individuals elected to those positions, and a participant&quot;s seat MUST
+      NOT be delegated to any other person.
+    </p>
+    <p>      
+      For most Chair nominees, the primary affiliation is their employer and
+      will match their affiliation in the W3C database. For contractors and
+      independents, this will normally be their contracting company or their
+      independent status; in some cases (e.g. where a consultant is consulting
+      for only one organization) this may be the organization for whom the
+      nominee is consulting.
+      <strong>Chair nominees, elected chairs, and appointed chairs MUST have
+      unique affiliations publicly stated at time of nomination, election (if it
+      has changed since last disclosure), or appointment.</strong>
```

Comment:
Can an chair nominee explicitly choose to run independent of their employer? i.e., they're aware that their affiliation and running for chair is not supported by their employer, and that they may be changing employers in the near future?

If so, language like "A chair nominee may choose to run independent of their current company, so long as their affiliation with that company changes within 6 months of election, otherwise they will be dismissed from their post and the chair position will automatically hold a new election for that role.

Reply:
I think the idea was that if one of the current chairs has affiliation X, no nominees or appointees should also have that same affiliation.  I would appreciate some kind of flexibility for people self-attesting their affiliation as "independent", but what's more valuable to be is the _recent disclosure_ about, say, current employment, substantial stock holdings, serving on Board, etc., because the group consensus _could well be_ to accept that nominee's/appointee's word that they're acting in an individual capacity, but should _not_ make such exceptions without all the facts fresh at hand.

Reply:
I don't like a "6 month" clause or whatever because then that still would give a bad actor 6 months to do chair-ish things while colluding with another chair in the interest of a shared organization (which is the issue we are trying to avoid here, right?). We wouldn't just throw out the 6 months of work done by the CG upon dismissing the institutionally-redundant chair, right? And yet that would be 6 months of "compromised" work.

Basically I think it's okay the way it is, and if someone wants to be Chair that badly and they currently work at the same company as another Chair, they can simply quit that job or resign that affiliation. Or wait until the next election.